aquarius/rbac.py

```diff
@@ -71,11 +71,14 @@ def validate_ddo_rbac(data):
         ).json()
 
     @staticmethod
-    def check_permission_rbac(event_type, address):
+    def check_permission_rbac(event_type, address, tx_id):
         payload = {
             "eventType": event_type,
             "component": "metadatacache",
-            "credentials": {"type": "address", "value": address},
+            "credentials": [
+                {"type": "address", "value": address},
+                {"type": "address", "value": tx_id},
```

Comment:
it should be 
```
 payload = {
            "eventType": event_type,
            "component": "metadatacache",
            "txid": tx_id,
            "credentials": {"type": "address", "value": address},
            "credentials": [
                {"type": "address", "value": address},
            ],
        }
 ```

Reply:
Thank you! That should be it.

Reply:
should aquarius also pass ddo as part of the rbac request?
so rbac can check the decrypted transaction sender vs ddo.nft.owner, or other additional checking

Reply:
I can add the DDO inside the payload for additional RBAC checking. What do you think @alexcos20?